plugins/inputs/dcos/README.md

```diff
@@ -0,0 +1,168 @@
+# DC/OS Input Plugin
+
+This input plugin gathers metrics from a DC/OS cluster.
+For more information, please check the [DC/OS Metrics](https://dcos.io/docs/1.9/metrics/) page.
+
+### Configuration:
+
+```toml
+# Input plugin for gathering DC/OS agent metrics
+[[inputs.dcos]]
+  # Hostname or ip address of DC/OS master for access from within DC/OS cluster
+  master_hostname=""
+  # Public URL of DC/OS cluster, e.g. http://dcos.example.com. Use of access from outside of the DC/OS cluster. master_hostname has higher priority, if set
+  #cluster_url=""
+  # Authentication token, obtained by running: dcos config show core.dcos_acs_token. Leave empty for no authentication.
+  # Warning: authentication token is valid only 5 days in DC/OS 1.10.
```

Comment:
Just to clarify my understanding, if using `master_hostname` then the `auth_token` is not required, and if using `cluster_url` then you would need to update this every 5 days?

Reply:
When using cluster_url  auth_token should set, cause it by default required, and yes, it will have to be updated each 5 days.  
Because of that I chose master_hostname as default, and left cluster_url for remote connection as second option.
Main reason is that I don't know how the plugin will be used. 

For sure, updating telegraf configuration each 5 days, because of invalid token, seems not usable, I can leave it in code and remove it from configuration.

Reply:
Okay, don't change it for now and I will figure out how we want to handle it.

Reply:
BTW I found the [POST /auth/login](https://docs.mesosphere.com/1.10/security/iam-api/#!/login/post_auth_login) endpoint in enterprise DC/OS (might need to scroll to the bottom).  It seems like it could be used, or perhaps another endpoint that is specific to the enterprise version, but without a copy it will be difficult to figure out.

Reply:
Good. As I hadn't available Enterprise DC/OS I didn't checked its API.

So, for Enterprise DC/OS we can have configuration for the service token and we can perform login. But for free DC/OS, this will not work.

Reply:
We will need an Enterprise DC/OS before we can implement any of it's login methods.  I did determine how to [create a jwt without expiration](https://medium.com/@richardgirges/authenticating-open-source-dc-os-with-third-party-services-125fa33a5add).

I think the required `auth_token_secret` is too sensitive to place in the telegraf config but we could document how to manually create the jwt and that could be added to the configuration.

Reply:
So, who can arrange a trial access to an Enterprise DC/OS instance? Should I ask Tim?

Reply:
The way  how to create a authetication token without expiration look as a hack depending on internals of DC/OS. 

Are we sure, that such stuff should be documented or referenced in an official open-source application?